.github/workflows/main.yml

```diff
@@ -0,0 +1,99 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  COLORTERM: 'yes'
+  TERM: 'xterm-256color'
+  PYTEST_ADDOPTS: '--color=yes'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ["2.7", "3.5", "3.6", "3.7"]
+        ansible:
+          - stable-2.7
+          - stable-2.8
+          - stable-2.9
+          - devel
+        exclude:
+          - python: "2.7"
+            ansible: "stable-2.7"
+          - python: "2.7"
+            ansible: "stable-2.8"
+          - python: "2.7"
+            ansible: "stable-2.9"
+          - python: "3.5"
+            ansible: "stable-2.7"
+          - python: "3.5"
+            ansible: "stable-2.8"
+          - python: "3.5"
+            ansible: "stable-2.9"
+          - python: "3.6"
+            ansible: "stable-2.7"
+          - python: "3.6"
+            ansible: "stable-2.8"
+          - python: "3.6"
+            ansible: "stable-2.9"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Install Ansible
+        run: pip install git+https://github.com/ansible/ansible.git@${{ matrix.ansible }}#egg=ansible
+      - name: Install dependencies
+        run: make test-setup
+      - name: Install Ansible devel specific dependencies
+        run: ansible-galaxy collection install community.general
+        if: matrix.ansible == 'devel'
+      - name: Run basic tests
+        run: make test-crud test-other
+      - name: Run Ansible devel specific tests
+        run: make SANITY_OPTS="--local" PYTHON_VERSION=${{ matrix.python }} sanity dist-test
+        if: matrix.ansible == 'devel'
+
+  checkmode:
```

Comment:
Not sure this is a thing if we don't have modules here.

Reply:
I think we will in the end, so I debated whether to leave it for when that time comes or just be ready for it.